test/e2e/network/service.go

```diff
@@ -2215,20 +2215,23 @@ var _ = common.SIGDescribe("Services", func() {
 		serviceName := "svc-itp"
 		ns := f.Namespace.Name
 		servicePort := 80
+		// If the pod can't bind to this port, it will fail to start, and it will fail the test,
+		// because is using hostNetwork. Using a not common port will reduce this possibility.
+		endpointPort := 10180
```

Comment:
I think a random port number is best here

Reply:
we discussed that approach before and that was discarded because the test failures will be hard to debug, you can't know which port is free in advance, at least, with this  approach you know that if it fails to bind it will fail consistently in the same port
This is not great, but the alternative is not much better